server/controllers/appointmentController.js

```diff
@@ -3,26 +3,72 @@ const mongoose = require("mongoose");
 
 exports.getLoggedInUserAppointments = async function (req, res) {
   try {
-    var query =  {meetingId: req.param.meetingId} ;
+    var query = { meetingId: req.params.meetingId };
```

Comment:
I thought the meeting will be just 1 to 3.
we can't assign many meeting Ids to users. Because each id will correspond with meeting durations like 15, 30, 45. 
You need to use logged in user id to get the list of appointments for logged in user.
Let me double-check this part.

Reply:
nvm, you are right,
Just recommend you write validation code to another file!

Reply:
Thanks we should still confirm it with Bonnie and then merge this.